plugins/mumble_plugin_utils.h

```diff
@@ -17,7 +17,13 @@
 // allows the string to be safely used
 // when constructing JSON documents via
 // string concatenation.
-static void escape(char *str) {
+//
+// Finally, escape ensures that the given
+// string is NUL-terminated by always
+// setting the last byte of the input
+// string to the value 0.
+static void escape(char *str, size_t size) {
+    str[size-1] = 0; // NUL terminate queried C strings. We do this to ensure the strings from the game are NUL terminated. They should be already, but we can't take any chances.
```

Comment:
This comment is no longer up-to-date.

How about just:

    // Ensure the input string is properly NUL-terminated.
    str[size-1] = 0;